server/app/views/admin/programs/predicates/EditConditionPartial.html

```diff
@@ -1,5 +1,5 @@
 <!--/*@thymesVar id="model" type="views.admin.programs.predicates.EditConditionPartialViewModel"*/-->
 
 <div
-  th:replace="~{admin/programs/predicates/EditConditionFragment :: editCondition(${model.conditionId}, ${model.hxEditConditionEndpoint()}, ${model.hxEditSubconditionEndpoint()}, ${model.operatorNodeTypes()}, ${model.questionOptions}, ${model.scalarOptions}, ${model.operatorOptions})}"
+  th:replace="~{admin/programs/predicates/EditConditionFragment :: editCondition(${model.conditionId}, ${model.hxEditConditionEndpoint()}, ${model.hxEditSubconditionEndpoint()}, ${model.selectedQuestionType()}, ${model.operatorNodeTypes()}, ${model.questionOptions}, ${model.scalarOptions}, ${model.operatorOptions}, ${model.valueOptions})}"
```

Comment:
(non-blocking) The parameters here are getting pretty out of control when it looks like the model can just be passed in. In a separate PR can you clean this up some.